db/migrate/20160311163203_change_data_type_for_column_vendor_in_vms_table_to_citext.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeDataTypeForColumnVendorInVmsTableToCitext < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :vendor, :citext
```

Comment:
yea, pglogical is optional because some customers won't want to do replication.
But everyone wants case insensitive search.
So this would become a hard requirement.

But Mac (homebrew), and Ubuntu have this contrib extension installed out of the box and Red Hat has it in `rh-postgresql94-postgresql-contrib`, which is part of the out of the box RHEL cd.

It seems like a reasonable requirement